t/t0004-status-query.t

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+
+test_description='Test Powerman status query'
```

Comment:
not necessarily for this PR, but do you need to cover the "pm -q foo[1-3]" case too?  Get all status and then fitler it down?  TBH, I didn't even know about the `-Q` option, I always do `pm -q foo[1-3]` :-)

Reply:
Yeah might as well!

I'm getting reacquainted with the project too.  The tests before were super inscrutable and now they at least tell you kind of what they are doing and can be more easily grouped functionally.